### DISCUSSION TITLE
Av1 Support on 7900XT

### DISCUSSION THREAD

**Discussion by PSIboy** [RandomEngy/VidCoder#1150] [Ideas]
Hello I was wondering when AV1 encode support for my new Radeon 7900xt (which has hardware AV1 suppport) is coming to Vidcoder?

**Comment by RandomEngy** [comment]
VidCoder exposes the functionality of HandBrake. To get that feature, you'd need to ask the [HandBrake team](https://github.com/HandBrake/HandBrake/issues/new/choose) to add it.
